parser-typechecker/src/Unison/Runtime/Interface.hs

```diff
@@ -42,7 +42,7 @@ import Data.Set as Set
     (\\),
   )
 import qualified Data.Set as Set
-import Data.Text (Text, isPrefixOf, pack)
+import Data.Text (Text, isPrefixOf)
```

Comment:
You can largely ignore this file, it got re-formatted due to differences in `ormolu` versions (which Dan and I have now negotiated haha). The only change is that the UCM version is now passed as `Text` rather than `String`.